src/components/tooltip/daikin-tooltip.ts

```diff
@@ -193,20 +205,19 @@ export class DaikinTooltip extends LitElement {
       >
         <slot
           @click=${this._handleClick}
-          @mouseenter=${this._handleMouseEnter}
-          @mouseleave=${this._handleMouseLeave}
           @focusin=${this._handleFocusIn}
           @focusout=${this._handleFocusOut}
+          @mouseenter=${this._handleMouseEnter}
+          @mouseleave=${this._handleMouseLeave}
         ></slot>
       </div>
       <span
```

Comment:
It would be good to provide a css part that allows users to customize the size of the tooltip.

```suggestion
      <span
        part="tooltip"
```